### DISCUSSION TITLE
Not sure what to do on Zonal notebook 41 section error

### DISCUSSION THREAD

**Discussion by VictoriAragon** [earthlab-education/Earth-Analytics-AY24#632] [Q&A]
![image](https://github.com/user-attachments/assets/383cadc1-3ea2-4418-ba8d-e982f0da06e9)

**Comment by eculler** [comment]
Hi @VictoriAragon -- this error is saying that `den_ndvi_zstats` is a tuple, and `.set_index()` is only available for `DataFrame`s. You should get a `DataFrame` out of the zonal stats function and not a tuple...but I can't see where you define that so it's hard to diagnose the problem.
